appveyor.yml

```diff
@@ -125,6 +125,9 @@ install:
   - copy C:\Python27\python.exe C:\Python27\python2.7.exe
   - set PATH=C:\Python27;%PATH%
 
+  # Run unit tests for bootstrap.py
+  - python src\bootstrap\bootstrap_test.py
```

Comment:
Wherever it happens, it should probably be in the same place in both travis and appveyor. My personal preference it to put it in the "test" or "script" or whatever phase just for clarity.